### PR TITLE
💄Tooltip show automatically

### LINKED ISSUE #1000: 💄 RulesGPT link improvements
Cc: @adamcogan @jaydenalchin @tiagov8 

Hi @Aibono1225 

We want to make the RulesGPT link more noticable.

### Tasks
- [x] Make the tooltip show automatically
- [x] Tooltip - Change text to “✨ Try out RulesGPT”
- [x] Appear after 3 seconds, disappear after 15
- [x] New rule button - Change to “Create an SSW Rule”
  image

**Figure: New rule button**

- [x] Help button - Change to “SSW Rules wiki” image

**Figure: Help button**

- [x] 📈 Analytics - Can we see how many people are clicking the RuleGPT button?

Thanks!

### PR DESCRIPTION
**Pull request by Aibono1225**
Fixes #1000 

Changes I made:
- Make the "RulesGPT" tooltip appear automatically after 3s, and disappear after 15s.
- Tooltip - Change text to “✨ Try out RulesGPT”
- New rule button - Change to “Create a SSW Rule”

D846D27E-0AE8-4BE9-9EA1-71C3AC17DEBD

### CODE REVIEW

File: src/components/header/header.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ const Header = ({ displayActions }) => {
               </a>
             </Tooltip>
 
-            <Tooltip text="Add New Rule">
+            <Tooltip text="Create a SSW Rule">
```
Comment (bradystroud) [inline src/components/header/header.js:61]:
```suggestion
            <Tooltip text="Create an SSW Rule">
```

File: src/components/tooltip/tooltip.js
Comment (bradystroud) [inline src/components/tooltip/tooltip.js:1]: Great job on the Tooltip! 👏 Here are a couple of quick ideas:

	1.	Adding delay props: How about we add showDelay and hideDelay as props? 🔄 It’d give us more wiggle room and we can say bye-bye to the text checking. 👋
	2.	location.pathname bit: It’s kinda just chilling in there but we’re not using it. 🤷‍♂️ Might be good to take it out to keep things clean and tidy. 🧹

Overall, fantastic work, keep it up! 🚀

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: src/components/tooltip/tooltip.js
Comment (Aibono1225) [reply src/components/tooltip/tooltip.js:1]: All Done ⭐
Added showDelay and hideDelay props.
And removed location.pathname

**Review by bradystroud — APPROVED** [review]
lgtm 🚀

**Pull request merged**